.github/workflows/ci-cd.yml

```
@@ -1,323 +1,372 @@
+      # TODO: Coverage report download temporarily disabled
+      # - name: Download coverage report
+      #   uses: actions/download-artifact@v4
+      #   with:
+      #     name: coverage-report
+      #     path: docs/
+
+      - name: Generate HTML documentation
+        run: |
+          cd openapi
+          npm run docs:generate
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)